src/tests/databases/mock_db/table.sql

```diff
@@ -1,17 +1,15 @@
 CREATE TABLE `gibberish` (
-    `id`      INT NOT NULL,
-    `grp`     VARCHAR(20) DEFAULT NULL,
-    `value`   INT DEFAULT NULL,
-    PRIMARY KEY (`id`, `grp`),
-    KEY (`id`)
+    `id`    INT NOT NULL,
```

Comment:
Should be `INTEGER`, not `INT` (to meet the same nomenclature as the next table).